python/app/model_entity_listing.py

```diff
@@ -91,7 +91,12 @@ def is_highlighted(self, model_index):
         return False
 
     def load_data(
-        self, sg_location, additional_fields=None, sort_field=None, direction="desc"
+        self,
+        sg_location,
+        additional_fields=None,
+        sort_field=None,
+        direction="desc",
+        filters=[],
```

Comment:
We should avoid defining default arguments are mutable values. Unwanted behavior can happen when you call the function and the `filters` argument is modified to, say, the value `x`. The next time you call the function and if you do not pass the `filters` parameter, it will still have the value `x`, instead of it being an empty list (as you probably wanted).
```suggestion
        filters=None,
```

Reply:
Wow how did I not know this, or maybe I did and I had forgotten. I'm quite rusty on Python these days.
But good shout thank you for calling it out.